client/packages/invoices/src/Returns/DetailView/DetailView.tsx

```diff
@@ -88,7 +92,7 @@ export const DetailView: FC = () => {
           )} */}
 
           {/* <Toolbar /> */}
-          {/* <DetailTabs tabs={tabs} /> */}
+          <DetailTabs tabs={tabs} />
           {/* <Footer /> */}
           {/* <SidePanel /> */}
         </TableProvider>
```

Comment:
haha I can't highlight it to put a comment there, but should also change the translation strings below `shipment` -> `return`

Reply:
Oh actually I'm always getting this error modal, even when the return does exist! I think it's because you have the display logic `data ? <>[table]</> : <AlertModal />`, and on the initial render data is undefined... probably since you added the type filter thing in the api file?